### DISCUSSION TITLE
12DF76D8-7C06-4B0A-9380-2504BC9719D9

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#155] [General]
# 12DF76D8-7C06-4B0A-9380-2504BC9719D9

Hoth is a remote ice planet in the Outer Rim, where it served as the main Rebel Base after they were discovered in Yavin IV by the Empire until 3 ABY

https://celestia.mobi/addon?item=12DF76D8-7C06-4B0A-9380-2504BC9719D9

**Comment by exocritic5457** [comment]
How was Hoth listed as orbiting every 526 days in Wookieepedia if it's around a B-class star and its greatest orbital distance at 526 days would still boil it... 
I'm still focused on this problem. it's a much larger distance here, of course, I should've assumed this franchise doesn't like hard science from the get-go at this point

**Reply by DaveBowman2001** [reply]
Star Wars was never a hard science fiction saga in the first place, but tbf to the creators, Hoth and every other planet introduced in the original trilogy were conceived long before the first real exoplanets were discovered. So in order for some planets to be quite habitable in real life based on the star type of their hosts, we have to make some orbital adjustments to make it happen

**Reply by DaveBowman2001** [reply]
Also, unlike Coruscant (which had a yellowish color to disprove it's a B-type), expanded literature (and its Wiki page) clearly state Hoth's sun is more blue-white in appearance

**Comment by exocritic5457** [comment]
Yeah I really shouldn't do this, even though I think the franchise is cool. science kinda just fell apart, more than when Palpatine "somehow returned"
